tactic/chain.lean

```diff
@@ -137,15 +137,10 @@ do tgt ← target,
    trace format!"new target: {tgt}",
    pure r
 
-private meta def chain_handle_trace (tactics : list (tactic α)) : tactic (list string) :=
+meta def chain (tactics : list (tactic α)) : tactic (list string) :=
 if is_trace_enabled_for `chain then
   chain_core (tactics.map trace_output)
 else 
   chain_core tactics
 
-meta def chain (tactics : list (tactic α)) : tactic (list string) :=
```

Comment:
There was no need to check `sequence.empty` here, because the same check is done in `chain_core`.